Source/carthage/Outdated.swift

```diff
@@ -54,13 +55,40 @@ public struct OutdatedCommand: CommandProtocol {
 
 				if !outdatedDependencies.isEmpty {
 					carthage.println(formatting.path("The following dependencies are outdated:"))
+
 					for (project, current, updated, latest) in outdatedDependencies {
+						let versionColor: Color.Named.Color
+						switch (current, updated, latest) {
+						case (_, updated, latest) where updated == latest:
+							// Update available and applicable
+							versionColor = .green
+						case (current, updated, latest) where current != updated && updated != latest:
+							// Update availabe and applicable, but not to the latest version due to version lock
+							versionColor = .yellow
+						case (current, updated, latest) where current == updated:
+							// Update available, but not applicable due to version lock
+							versionColor = .red
+						default:
+							versionColor = .white
+						}
+
 						if options.outputXcodeWarnings {
-							carthage.println("warning: \(formatting.projectName(project.name)) is out of date (\(current) -> \(updated)) (Latest: \(latest))")
+							carthage.println("warning: \(formatting.projectName(project.name)) is out of date (\(current) -> \(updated) (Latest: \(latest))_")
```

Comment:
Why did this change from `) (Latest` to ` (Latest`?

Reply:
I think that was unintentional, will revert